docs/user_guide/reporting/Supervisor Notifications

```diff
@@ -0,0 +1,22 @@
+grand_parent: User Guide
```

Comment:
Is there a reason you are creating a new page here vs editing this page https://relativitydev.github.io/relativity-trace-documentation/docs/user_guide/reporting/notifications.html